src/types/products.ts

```diff
@@ -131,6 +131,14 @@ interface RootDocsPath {
 	 * and will dynamically account for version context.
 	 */
 	addOverviewItem?: boolean
+
+	/**
+	 * An optional description for this category of documentation.
+	 * Shown as the subtitle of the docs landing hero element.
+	 * If omitted, falls back to the page's authored frontMatter.description,
+	 * or falls back to an empty string.
+	 */
+	description?: string
```

Comment:
Have made this `description` property, and am falling back to `frontMatter.description`, based on [input from Judith in Slack](https://hashicorp.slack.com/archives/C01KCU4HDPY/p1661181737637109?thread_ts=1660952605.279929&cid=C01KCU4HDPY) that authors will likely want to use `frontMatter.description`.